source/beamline/AMPVControl.cpp

```diff
@@ -209,6 +209,13 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 	return NoFailure;
 }
 
+// Start a move to the value setpoint:
+AMControl::FailureExplanation AMPVControl::move(const QString &stringSetPoint)
```

Comment:
Could you explain why this is required?  It also is not very safe since it doesn't check anything, nor does it listen to any potential preset parameters.